### ISSUE TITLE
confusing when training my own data

### ISSUE THREAD

**Issue by TankosTao** [open-mmlab/mmocr#886]
Thanks for your error report and we appreciate it a lot.

**Describe the bug**
when i Train my own dataset with dbnet,(dataset_type = IcdarDataset) i got some unexpected log info as follows: 

2022-03-27 18:00:42,086 - mmocr - INFO - Epoch [1][5/500]       lr: 7.000e-03, eta: 13 days, 21:15:38, time: 2.000, data_time: 1.734, memory: 6690, loss_prob: 0.0000, loss_db: 1.0000, loss_thr: 0.0000, loss: 1.0000
2022-03-27 18:00:45,054 - mmocr - INFO - Epoch [1][10/500]      lr: 7.000e-03, eta: 9 days, 0:12:52, time: 0.595, data_time: 0.304, memory: 6690, loss_prob: 0.0000, loss_db: 1.0000, loss_thr: 0.0000, loss: 1.0000
2022-03-27 18:00:47,030 - mmocr - INFO - Epoch [1][15/500]      lr: 7.000e-03, eta: 6 days, 22:05:23, time: 0.395, data_time: 0.117, memory: 6690, loss_prob: 0.0000, loss_db: 1.0000, loss_thr: 0.0000, loss: 1.0000
2022-03-27 18:00:50,551 - mmocr - INFO - Epoch [1][20/500]      lr: 7.000e-03, eta: 6 days, 9:48:17, time: 0.702, data_time: 0.401, memory: 6690, loss_prob: 0.0000, loss_db: 1.0000, loss_thr: 0.0000, loss: 1.0000
2022-03-27 18:00:52,478 - mmocr - INFO - Epoch [1][25/500]      lr: 7.000e-03, eta: 5 days, 15:56:15, time: 0.387, data_time: 0.114, memory: 6690, loss_prob: 0.0000, loss_db: 1.0000, loss_thr: 0.0000, loss: 1.0000
2022-03-27 18:00:55,320 - mmocr - INFO - Epoch [1][30/500]      lr: 7.000e-03, eta: 5 days, 9:05:28, time: 0.569, data_time: 0.330, memory: 6690, loss_prob: 0.0000, loss_db: 1.0000, loss_thr: 0.0000, loss: 1.0000
**Reproduction**

1. What command or script did you run?
python tools/train.py configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py --work-dir dbnet

2. Did you make any modifications on the code or config? Did you understand what you have modified?
No i only modify the /mmocr/configs/_base_/det_datasets/icdar2015.py file

**Environment**

sys.platform: linux
Python: 3.7.3 (default, Mar 27 2019, 22:11:17) [GCC 7.3.0]
CUDA available: True
GPU 0: GeForce RTX 3090
CUDA_HOME: /usr/local/cuda
NVCC: Build cuda_11.2.r11.2/compiler.29373293_0
GCC: gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
PyTorch: 1.9.0+cu111
PyTorch compiling details: PyTorch built with:
  - GCC 7.3
  - C++ Version: 201402
  - Intel(R) Math Kernel Library Version 2020.0.0 Product Build 20191122 for Intel(R) 64 architecture applications
  - Intel(R) MKL-DNN v2.1.2 (Git Hash 98be7e8afa711dc9b66c8ff3504129cb82013cdb)
  - OpenMP 201511 (a.k.a. OpenMP 4.5)
  - NNPACK is enabled
  - CPU capability usage: AVX2
  - CUDA Runtime 11.1
  - NVCC architecture flags: -gencode;arch=compute_37,code=sm_37;-gencode;arch=compute_50,code=sm_50;-gencode;arch=compute_60,code=sm_60;-gencode;arch=compute_70,code=sm_70;-gencode;arch=compute_75,code=sm_75;-gencode;arch=compute_80,code=sm_80;-gencode;arch=compute_86,code=sm_86
  - CuDNN 8.0.5
  - Magma 2.5.2
  - Build settings: BLAS_INFO=mkl, BUILD_TYPE=Release, CUDA_VERSION=11.1, CUDNN_VERSION=8.0.5, CXX_COMPILER=/opt/rh/devtoolset-7/root/usr/bin/c++, CXX_FLAGS= -Wno-deprecated -fvisibility-inlines-hidden -DUSE_PTHREADPOOL -fopenmp -DNDEBUG -DUSE_KINETO -DUSE_FBGEMM -DUSE_QNNPACK -DUSE_PYTORCH_QNNPACK -DUSE_XNNPACK -DSYMBOLICATE_MOBILE_DEBUG_HANDLE -O2 -fPIC -Wno-narrowing -Wall -Wextra -Werror=return-type -Wno-missing-field-initializers -Wno-type-limits -Wno-array-bounds -Wno-unknown-pragmas -Wno-sign-compare -Wno-unused-parameter -Wno-unused-variable -Wno-unused-function -Wno-unused-result -Wno-unused-local-typedefs -Wno-strict-overflow -Wno-strict-aliasing -Wno-error=deprecated-declarations -Wno-stringop-overflow -Wno-psabi -Wno-error=pedantic -Wno-error=redundant-decls -Wno-error=old-style-cast -fdiagnostics-color=always -faligned-new -Wno-unused-but-set-variable -Wno-maybe-uninitialized -fno-math-errno -fno-trapping-math -Werror=format -Wno-stringop-overflow, LAPACK_INFO=mkl, PERF_WITH_AVX=1, PERF_WITH_AVX2=1, PERF_WITH_AVX512=1, TORCH_VERSION=1.9.0, USE_CUDA=ON, USE_CUDNN=ON, USE_EXCEPTION_PTR=1, USE_GFLAGS=OFF, USE_GLOG=OFF, USE_MKL=ON, USE_MKLDNN=ON, USE_MPI=OFF, USE_NCCL=ON, USE_NNPACK=ON, USE_OPENMP=ON, 

TorchVision: 0.10.0+cu102
OpenCV: 4.2.0
MMCV: 1.4.6
MMCV Compiler: GCC 7.3
MMCV CUDA Compiler: 11.1
MMOCR: 0.4.1+7c90936

**Comment by xinke-wang** [comment]
Hi, this may happen because of incorrect data annotations. Pls double-check the customized dataset, for example, visualize the converted annotation files by drawing the bounding boxes on the images, and see if there are any unexpected issues.

**Comment by TankosTao** [comment]
> Hi, this may happen because of incorrect data annotations. Pls double-check the customized dataset, for example, visualize the converted annotation files by drawing the bounding boxes on the images, and see if there are any unexpected issues.

I followed https://github.com/open-mmlab/mmocr/pull/821 
tools/data/textdet/vintext_converter.py in line 117
            x_min = min(bbox[0], bbox[2], bbox[4], bbox[6])
            x_max = max(bbox[0], bbox[2], bbox[4], bbox[6])
            y_min = min(bbox[1], bbox[3], bbox[5], bbox[7])
            y_max = min(bbox[1], bbox[3], bbox[5], bbox[7])
y_max should = max() which make the area of text region = 0
cased the error

**Comment by xinke-wang** [comment]
> > Hi, this may happen because of incorrect data annotations. Pls double-check the customized dataset, for example, visualize the converted annotation files by drawing the bounding boxes on the images, and see if there are any unexpected issues.
> 
> I followed #821 tools/data/textdet/vintext_converter.py in line 117 x_min = min(bbox[0], bbox[2], bbox[4], bbox[6]) x_max = max(bbox[0], bbox[2], bbox[4], bbox[6]) y_min = min(bbox[1], bbox[3], bbox[5], bbox[7]) y_max = min(bbox[1], bbox[3], bbox[5], bbox[7]) y_max should = max() which make the area of text region = 0 cased the error

This PR #821 has not been fully reviewed, thank you for pointing out the bug! @Mountchicken pls fix this.
